File: biggie_size.py
arr = [-1,5,17,-32,0]

def biggie_size2(lst):
	for i in lst:
		if i > 0:
			i = "BIG"
		print i

print biggie_size2(arr)

# Challenge: Given an array, write a function that changes all positive numbers in the array to "BIG".

File: biggie_size.js
var arr = [-1, 5, 17, -32, 0]

function biggieSize(array) {
	for (var i = 0; i < array.length; i++) {
		if (array[i] > 0) {array[i] = "BIG"}
	}
	return array;
} 

console.log(biggieSize(arr));



// Challenge: Given an array, write a function that changes all positive numbers in the array to "BIG".